artifactory/commands/golang/gopublish.go

```diff
@@ -17,6 +17,7 @@ type GoPublishCommandArgs struct {
 	buildConfiguration *utils.BuildConfiguration
 	version            string
 	detailedSummary    bool
+	excludedPatterns   []string
```

Comment:
excludedPatterns --> exclusions
Let's also modify all functions and variables throughout the code accordingly.